tests/deploy_test.go

```diff
@@ -6,19 +6,19 @@ import (
 	"strings"
 	"testing"
 
-	types "github.com/openfaas/faas-provider/types"
+	faasSDK "github.com/openfaas/faas-cli/proxy"
```

Comment:
I think i would be ok with just calling it `sdk` or `proxy`, my fingers stumble writing `faasSDK`

Reply:
updated to `sdk`